src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -602,7 +671,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IBinaryKind As BinaryOperationKind Implements IBinaryOperatorExpression.BinaryKind
```

Comment:
I'll change all the cases of this convention to the Interface_Member convention that @CyrusNajmabadi suggested in an earlier PR comment, after Gen merges this PR.